src/main/java/de/teamlapen/vampirism/entity/ai/VampireAIBiteNearbyEntity.java

```diff
@@ -45,7 +45,7 @@ public boolean shouldContinueExecuting() {
     @Override
     public boolean shouldExecute() {
         if (vampire.wantsBlood()) {
-            List<EntityCreature> list = vampireEntity.getEntityWorld().getEntitiesWithinAABB(EntityCreature.class, getBiteBoundingBox());
+            List<EntityCreature> list = vampireEntity.getEntityWorld().getEntitiesWithinAABB(EntityCreature.class, getBiteBoundingBox(), (entityCreature -> !(entityCreature instanceof IVampireMob)));
```

Comment:
This prevents the crash, but does not resolve the root issue.
Even IVampireMobs should have a ExtendedCreature capability shouldn't they? (Even though it is not used)

Reply:
I have tested it with multiple tries and creature entities and it has never throws an error. So this works until someone finds the basic problem or gets the IllegalStateException with creatures other than vampires

Reply:
The exception will not be thrown the first time get() is called in ExtendedCreature, but it will be triggered randomly